server/src/immich/api-v1/asset/asset.core.ts

```diff
@@ -48,6 +48,7 @@ export class AssetCore {
       faces: [],
       sidecarPath: sidecarPath || null,
       isReadOnly: dto.isReadOnly ?? false,
+      stackParentId: null,
```

Comment:
We should not need this change, and it should not be required if it is. It should default to null at the database level when not provided.

Reply:
Removed. Thank you.